.github/CHANGELOG.md

```diff
@@ -153,6 +153,7 @@ rng = np.random.default_rng()
 random_mat1 = rng.random((3,2))
 random_mat2 = rng.standard_normal(3, requires_grad=False)
 ```
+* New ISWAP operation added to default_qubit device.
```

Comment:
```suggestion
* Added the ISWAP operation with support for the default_qubit device.
  [(#1298)](https://github.com/PennyLaneAI/pennylane/pull/1298)
```

Reply:
@brs96 could you add the link after the description here? :slightly_smiling_face: (No worries if you prefer the original short description)